packages/loader/src/ShaderChunkLoader.ts

```diff
@@ -12,7 +12,7 @@ import {
 
 @resourceLoader("ShaderChunk", ["glsl"])
 export class ShaderChunkLoader extends Loader<void[]> {
-  private static _shaderIncludeRegex = /\s#include\s+"([./][^\\"]+)"/gm;
+  private static _shaderIncludeRegex = /#include\s+"([./][^\\"]+)"/gm;
 
```

Comment:
Compatible with the case `#include` macro appear at the first line and first charactor place. It will not be matched before.